### ISSUE TITLE
dialog outside of the taskpane

### ISSUE THREAD

**Issue by nickersd** [OfficeDev/office-ui-fabric-react#8341]
I need a dialog that has more real estate than just the size of the task pane.  I've attached a screenshot from the API documents that show the size that I would need. 
I've tried doing this with css (ie: containerClassName: 'myDialogClass' in ModalProps) but there appears to be a limit to only the size of the taskpane.  Is there a different surface that I should be using? Is there something that I'm missing in the ModalProps? 
![overview-with-app-dialog](https://user-images.githubusercontent.com/4313261/54456753-ef96ff80-4735-11e9-9bbf-abb064fcb183.png)

**Comment by KevinTCoughlin** [comment]
The screenshot above looks like some sort of `Popup`-like component with custom `height` & `width` styling. I'll take a pass at putting a Codepen together to accomplish and get back to you shortly. It _should_ be possible with current controls (I think).

**Comment by KevinTCoughlin** [comment]
@nickersd, you can customize the `Dialog`'s min and max width, but its admittedly a bit tricky today. It requires overriding the media query that is being set.

Here's an example that bumps the `Dialog` width beyond its normal bounds using `merge-styles` approach: https://codepen.io/kevintcoughlin/pen/bQGQpp

See https://github.com/OfficeDev/office-ui-fabric-react/issues/5175#issuecomment-435632440 for context.

Let me know if you're trying to achieve something else and I've misread.

**Comment by nickersd** [comment]
Thanks, I've added the styles prop (had to upgrade to latest for that) but I'm still facing the issue.  I've attached a screenshot.  Note the "Manually tag the element" dialog on the right (with a bunch of x's to show that the title is wrapping.)  I have to create some more dialog's (not in the screenshot)  that will need more real estate. 
![dialogSS](https://user-images.githubusercontent.com/4313261/55081754-d1ef6180-5076-11e9-9c70-c509122c1d53.png)
